assets/components/tracking/connector/ConnectorForm.tsx

```diff
@@ -63,173 +63,190 @@ export function ConnectorForm({form, onCreate}: { form: FormInstance, onCreate:
             </Form.Item>
 
             {
-                provider === ConnectorProvider.OVH && <>
+                provider !== undefined && <>
                     {
-                        Object.keys(ovhFields).map(fieldName => <Form.Item
-                            key={ovhFields[fieldName as keyof typeof ovhFields]}
-                            label={ovhFields[fieldName as keyof typeof ovhFields]}
-                            name={['authData', fieldName]}
-                            rules={[{required: true, message: t`Required`}]}
-                        >
-                            <Input autoComplete='off'/>
-                        </Form.Item>)
+                        [ConnectorProvider.AutoDNS, ConnectorProvider['Name.com']].includes(provider) && <Alert
+                            message={t`This provider does not provide a list of supported TLD. Please double check if the domain you want to register is supported.`}
+                            type='warning'
+                            style={{marginBottom: '2vh'}}
```

Comment:
Viewport-relative height doesn't seem to make sense here. Should probably be in `em`